Box.V2/Models/BoxPreflightCheck.cs

```diff
@@ -18,11 +18,13 @@ public class BoxPreflightCheck
         /// The upload URL to optionally use when uploading the file
         /// </summary>
         [JsonProperty(PropertyName = FieldUploadUrl)]
+        [Obsolete]
```

Comment:
So UploadUri & UploadToken will be always null in the response?

Should we mark FieldUploadUrl & FieldUploadToken as [Obsolete] as well?

ref:
https://docs.box.com/reference#preflight-check